scripts/setup.sh

```diff
@@ -179,19 +205,56 @@ get_ext_files()
   done
 }
 
+get_activated_extensions_on_cluster() {
+  DYNATRACE_URL=$1
+  DYNATRACE_ACCESS_KEY=$2
+
+  if RESPONSE=$(curl -k -s "$DYNATRACE_URL/api/v2/extensions" -w "<<HTTP_CODE>>%{http_code}" -H "Accept: application/json; charset=utf-8" -H "Content-Type: application/json; charset=utf-8" -H "Authorization: Api-Token $DYNATRACE_ACCESS_KEY" --connect-timeout 20); then
+    CODE=$(sed -rn 's/.*<<HTTP_CODE>>(.*)$/\1/p' <<<"$RESPONSE")
+    EXTENSIONS_FROM_CLUSTER=$(echo "$RESPONSE" | sed -r 's/<<HTTP_CODE>>.*$//' | jq -r '.extensions[] | select(.extensionName) | "\(.extensionName):\(.version)"')
+  fi  
+}
+
+activate_extension_on_cluster() {
+  DYNATRACE_URL=$1
+  DYNATRACE_ACCESS_KEY=$2
+  EXTENSIONS_FROM_CLUSTER=$3
+
+  if [ -z "$EXTENSIONS_FROM_CLUSTER" ]; then
```

Comment:
something is wrong here... currently we will make upload only when EXTENSIONS_FROM_CLUSTER is empty